entries/entry03.md

```diff
@@ -1,7 +1,19 @@
 # Entry 3
```

Comment:
Sources: 0 (need multiple)
EDP: 1
Knowledge: 1 (next time, add more code snippets, and with backticks)
Skills: 0
Professionalism: 1